examples/rac-tailwind/package.json

```diff
@@ -2,14 +2,9 @@
   "name": "rac-tailwind-example",
   "private": true,
   "scripts": {
-    "start": "parcel src/index.html"
+    "start": "parcel src/index.html",
+    "build": "parcel build src/index.html"
   },
-  "workspaces": [
-    "../../packages/react-aria-components",
-    "../../packages/react-aria",
-    "../../packages/react-stately",
-    "../../packages/*/*"
-  ],
```

Comment:
These are nice for local development but would mean that in CI we don't use the published version. Not sure how we could have both...